src/test/java/spimdata/OmeZarrV4S3SpimDataTests.java

```diff
@@ -1,15 +1,26 @@
 package spimdata;
 
 import mpicbg.spim.data.SpimData;
+import mpicbg.spim.data.sequence.MultiResolutionSetupImgLoader;
+import net.imglib2.Cursor;
+import net.imglib2.RandomAccessibleInterval;
+import net.imglib2.type.numeric.NumericType;
+import net.imglib2.type.numeric.RealType;
+import net.imglib2.view.Views;
 import org.embl.mobie.io.ome.zarr.openers.OMEZarrS3Opener;
 import org.junit.jupiter.api.Test;
 
 import java.io.IOException;
+import java.util.Arrays;
 
-import static org.junit.jupiter.api.Assertions.*;
+import static org.junit.jupiter.api.Assertions.assertArrayEquals;
+import static org.junit.jupiter.api.Assertions.assertEquals;
 
-public class OmeZarrV4S3SpimDataTests {
+public class OmeZarrV4S3SpimDataTests < N extends NumericType< N > & RealType< N > >
```

Comment:
Should this class be removed and joined with the below `OmeZarrS3V4Opener`?

Reply:
no